samples/TestFtpServer/OptionsConfigSource.cs

```diff
@@ -81,7 +81,7 @@ public IConfigurationProvider Build(IConfigurationBuilder builder)
                 { "c|certificate=", "Set the SSL certificate", v => values["ftps:certificate"] = v },
                 { "k|private-key=", "Set the private key file for the SSL certificate", v => values["ftps:privateKey"] = v },
                 { "P|password=", "Password for the SSL certificate", v => values["ftps:password"] = v },
-                { "i|implicit", "Use implicit FTPS", v => values["ftps:implicit"] = XmlConvert.ToBoolean(v.ToLowerInvariant()) ? "true" : "false" },
+                { "i|implicit", "Use implicit FTPS", v => values["ftps:implicit"] = "true" },
```

Comment:
why is this bool option declaration different to all the others next to it?

Shouldn't it make use of `v`?

Following the other declarations it would be

`v != null ? "true" : "false"` instead of `"true"`

Reply:
It should allow both `True` and `true` and the value may come from the `appsettings.json`, so - with your change - it would enable this option even with the value `false`.

Reply:
I did not think of it as a config option. In my assumption, you can only provide it through command-line and if provides it then it means it is true, hence the true-only change